app/Main.hs

```diff
@@ -24,11 +24,14 @@ import qualified System.Posix.IO
 import qualified System.Posix.Terminal
```

Comment:
There is an unused `import Data.Monoid ((<>))` in this module. Would be nice to catch such warnings as compile errors